src/components/open-api-page/partials/operation-object/style.module.css

```diff
@@ -52,7 +57,8 @@
 
 .summary {
 	/* Base typography */
-	composes: g-type-body from global;
+	composes: g-type-body-200 from global;
```

Comment:
⛏️ Nit, maybe out of scope - should we update this to like `hds-typography-body-300` or similar? I think it'd have minimal to no difference, but would help root out "mktg" classes (which we might be able to remove once `dot-io` sites are in `hashicorp/web`, I think!)

```suggestion
	composes: hds-typography-body-300 from global;
```

| Type class | Preview |
| - | - |
| `g-type-body-200` | <img width="1570" alt="g-type-body-200" src="https://user-images.githubusercontent.com/4624598/225446532-6ab8b251-b337-4e70-a67f-3fcf7d174fa9.png"> |
| `hds-typography-body-300` | <img width="1564" alt="hds-typography-body-300" src="https://user-images.githubusercontent.com/4624598/225446538-f1ebecd3-d6f0-4269-af4e-9e0713c2bbc5.png"> |